setup.py

```diff
@@ -199,6 +199,7 @@ def setup_package():
                                  'Operating System :: MacOS',
                                 ],
                     cmdclass=cmdclass,
+                    setup_requires=['cython', f'numpy>={NUMPY_MIN_VERSION}'],
```

Comment:
py2.7 support
```suggestion
                    setup_requires=['cython', 'numpy>={}'.format(NUMPY_MIN_VERSION)],
```

Reply:
Python 2.7 is not supported and it's impractical to continue supporting.

https://www.python.org/doc/sunset-python-2/

Reply:
It appears this PR might help a good number of users aiming to employ lap.  If py2.7 must remain suppored, how can we resolve?  I'd be happy to help.